shoop/admin/templates/shoop/admin/contact_groups/edit.jinja

```diff
@@ -6,23 +6,15 @@
     {% call content_with_sidebar(content_id="contact_group_form") %}
         <form method="post" id="contact_group_form">
             {% csrf_token %}
-            {% call content_block(_("General information"), "fa-info-circle") %}
-                {{ language_dependent_content_tabs(form) }}
-                {{ render_monolingual_fields(form) }}
-            {% endcall %}
+            {% include form.form_defs["base"].template_name %}
             {% if contact_group.pk %}
-                {% call content_block(_("Members"), "fa-users") %}
-                    {# TODO: This is very bare-bones; should probably have UI to add/remove members etc... #}
-                    <ul>
-                    {% for member in contact_group.members.all() %}
-                        {% set u=shoop_admin.model_url(member, default="") %}
-                        <li>{% if u %}<a href="{{ u }}">{% endif %}{{ member }}{% if u %}</a>{% endif %}</li>
-                    {% else %}
-                        <li><i>{% trans %}No members.{% endtrans %}</i></li>
-                    {% endfor %}
-                    </ul>
-                {% endcall %}
+                {% include form.form_defs["members"].template_name %}
```

Comment:
After that for change for form defs you can move contact_group.pk check inside members template.